scripts/variantstore/wdl/GvsExtractCallset.wdl

```diff
@@ -64,18 +72,14 @@ workflow GvsExtractCallset {
                 excluded_intervals       = excluded_intervals,
                 emit_pls                 = emit_pls,
                 service_account_json     = service_account_json,
-                output_file              = "${output_file_base_name}_${i}.vcf.gz"
+                output_file              = "${output_file_base_name}_${i}.vcf.gz",
+                last_modified_timestamp  = GetLastModifiedDateOrUniqueValue.last_modified_timestamp
```

Comment:
just to make this extra clear, maybe `source_tables_last_modified_timestamp` or something?